src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -183,5 +184,12 @@ The .NET Foundation licenses this file to you under the MIT license.
     <PropertyGroup Condition="'$(TargetOS)' == 'osx' and '$(StripSymbols)' == 'true' and $(_DsymUtilOutput.Contains('--minimize'))">
       <DsymUtilOptions>$(DsymUtilOptions) --minimize</DsymUtilOptions>
     </PropertyGroup>
+
+    <Exec Command="echo &quot;int main(void){}&quot; | &quot;$(CppCompilerAndLinker)&quot; -fuse-ld=lld -xc -o /dev/null -" IgnoreExitCode="true" StandardOutputImportance="Low" StandardErrorImportance="low" Condition="'$(UseLLVMLinker)' == 'true'">
```

Comment:
Do we need to test this upfront? What is the compiler behavior when one specifies `-fuse-ld=lld` and LLVM linker is not present?

Reply:
I added it for consistency to match other upfront errors above. All those `<Error`s have some system-defined behavior if we remove them.

Reply:
The other errors are for basic scenarios where it is important to guide people to self-diagnose and fix the problem.

I see this one as advanced scenario. We do not bother to show custom errors for advanced scenarios.

Reply:
How are we making the basic and advanced distinction? e.g. symbol stripping cannot be considered as a basic scenario, toolchains do not provide symbol stripping tool with core compiler driver, it is an optional tool. We enable it with `StripSymbols=true` and if `objcopy` tool is not present on the PATH, we issue a custom error on line 170.

Reply:
Based on how many people we expect to use it. We know that a lot of people want to use symbol stripping. We have been asked about it many times.

Reply:
Thanks. FTR, my motivation for adding this option was to support cases like https://github.com/dotnet/runtime/pull/52244, where we switched our shipping binaries to compile with `lld` by default, when it is available. Moreover, there are some linux distros which provide llvm-toolchain (instead of gnu-toolchain+binutils) in their default build-essential like package, but in such environments `/usr/bin/ld` is a symlink to lld.

Reply:
> What is the compiler behavior when one specifies `-fuse-ld=lld` and LLVM linker is not present?

```
  Generating native code
clang : error : invalid linker name in argument '-fuse-ld=lld' [/runtime/src/coreclr/tools/aot/crossgen2/crossgen2.csproj]
```